interop/javascript.md

```diff
@@ -42,10 +51,22 @@ This is doing three important things:
 
 So now we can set Elm up in any `<div>` we want. So if you are using React, you can create a component that just sets this kind of thing up. If you are using Angular or Ember or something else, it should not be too crazy either. Just take over a `<div>`.
 
-The next section gets into how to get your Elm and JavaScript code to communicate with each other in a nice way.
+### Fullscreen App
+
+If you don't need the app to be tied to a specific node, you can also get an elm app object by using the fullscreen method:
+
+```html
+<div id="main"></div>
```

Comment:
As a beginner, I'm not sure if this div is needed here for fullscreen apps. I included it because all examples I have seen have at least one element in the body besides the script. But you should remove it if it is unneeded.